pkg/store/sandbox/status.go

```diff
@@ -53,23 +53,17 @@ import (
 // +-------------> DELETED
 
 // State is the sandbox state we use in containerd/cri.
-// It includes init and unknown, which are internal states not defined in CRI.
+// It includes unknown, which are internal states not defined in CRI.
```

Comment:
/s/are/is/

which is an internal state not defined in CRI.. (INIT state has been removed)

Reply:
Done.